torchx/README.md

```diff
@@ -59,7 +59,13 @@ Once downloaded, we will compile `Torchx` bindings. You will need `make`/`nmake`
 - [Microsoft Visual C++ 2019 Redistributable](https://visualstudio.microsoft.com/downloads/)
 - [CMake](https://cmake.org/)
 
-Currently there are no precompiled versions available for Apple M1. [See this issue
+For Apple M1-series, you can download precompiled LibTorch binaries if you are using HomeBrew,
+```shell
+brew install libtorch
+export LIBTORCH_DIR="$(brew --cellar libtorch)/$(brew list --versions libtorch | tr ' ' '\n' | tail -1)"
+```
+
+Currently there are no other precompiled versions available for Apple M1. [See this issue
```

Comment:
Yeah, that makes sense.
Perhaps we can just emphasize this so:

"Currently, there are only precompiled versions for M1 via HomeBrew"

instead of "no other versions"